src/Tests/TestHelperTrait.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Nodes\Api\Tests;
+
+use App;
+use Nodes\Api\Auth\Auth;
+use PHPUnit_Framework_MockObject_MockObject;
+
+/**
+ * Api controllers tests helper.
+ *
+ * Use within tests of api controllers to call api routes as authenticated user, or as a guest.
+ *
+ * Here is an inline example:
+ * <code>
+ * <?php
+ * $this->being($user)->callApi('GET', '/api/users')->seeJson((new UserTransformer)->transform($user));
+ * ?>
+ * </code>
+ */
+trait TestHelperTrait
+{
+    /**
+     * @var PHPUnit_Framework_MockObject_MockObject|Auth
+     */
+    protected $authMock;
+
+    /**
+     * Act as provided user.
+     *
+     * This simulates user being authenticated with token.
+     *
+     * User does not need to be saved to database.
+     *
+     * @param $user
+     * @return $this
+     */
+    public function being($user)
+    {
+        $this->authMock = $this->getMockBuilder(Auth::class)->disableOriginalConstructor()->getMock();
+        $this->authMock->method('user')->willReturn($user);
+        App::instance('api.auth', $this->authMock);
+        return $this;
+    }
+
+    /**
+     * Visit the given URI with a JSON request.
+     *
+     * Api call will contain 'Accept' header required by nodes api package.
+     *
+     * To call api as authenticated user chain being() method first.
+     *
+     * @param  string $method
+     * @param  string $uri
+     * @param  array $data
+     * @param  array $headers
+     * @return $this
+     */
+    public function callApi($method, $uri, array $data = [], array $headers = [])
+    {
+        return $this->json($method, $uri, $data,
+            array_merge(['Accept' => 'application/vnd.nodes.v' . $this->getApiVersion() . '+json'], $headers));
```

Comment:
@rugaard That was intended ;)

I can't think of a case where we'll need to override 'Accept' header, but also not really about any potential problems with it. On incorrect 'Accept' header tests are just failing, so helper user would spot that immediately.

If this does not convince you, than just change the order of arguments in array_merge() - that should prevent 'Accept' from being overridden, while still allowing custom headers to be used.